sizing.tex

```diff
@@ -1,87 +1,19 @@
 \section{Sizing}\label{sizing}
 
 A fundamental question is how large the LSST Data Management System must be. To
-this end, a complex analytical model has been developed driven by input from
+this end, an analytical model has been developed driven by input from
 the requirements specifications. Specifications from the science requirements
 and other subsystem designs, and the observing strategy, translate directly
-into numbers of detected sources and astronomical objects, and ultimately into
-required network bandwidths and the size of storage systems. Specific science
-requirements of the survey determine the data quality that must be maintained
-in the DMS products, which in turn determine the algorithmic requirements and
-the computer power necessary to execute them. The relationship of the elements
-of this model and their flow-down from systems and DMS requirements is shown in
-Figure \ref{fig:sizing-model}. Detailed sizing computations and associated
-explanations appear in LSST Documents listed on the Figure.
-
-\begin{figure}
-\centering
-\includegraphics[width=\textwidth]{images/SizingModel.pdf}
-\caption{DMS Infrastructure Sizing and Estimation.}
-\label{fig:sizing-model}
-% Silently cite documents mentioned in diagram
-\nocite{LDM-148,LDM-151,LDM-152,LDM-135,LDM-138,LDM-140,LDM-129}
-\nocite{LSE-81,LSE-82,LDM-141,LDM-139,LDM-144,LDM-143}
-\nocite{LPM-17,LSE-29,LSE-30,LSE-61,LSE-163}
-\nocite{Document-5373,LDM-142,LSE-78}
```

Comment:
How many of these documents should be formally deprecated?

(Not necessarily on this ticket, but this is a good opportunity to note them).

Reply:
I guess LDM-141, 139, 144, 143 (and 138/140) could be demoted to "Document" status.  They're still valuable, but they no longer control the design of the system.

Reply:
Submitted RFC to capture this.